redisq/queue.go

```diff
@@ -90,21 +91,31 @@ func NewQueue(opt *taskq.QueueOptions) *Queue {
 		schedulerLockPrefix: redisPrefix + opt.Name + ":scheduler-lock:",
 	}
 
+	schedulerDeferFunc := func(name string, wg *sync.WaitGroup) {
+		wg.Done()
+		// Recover from panic. but the panic goroutine is not working anymore
+		// TODO create an extra scheduler once panic happend
+		if err := recover(); err != nil {
+			stack := string(debug.Stack())
+			internal.Logger.Printf("%v,panic recovered: %v,stack=%s", name, err, stack)
+		}
+	}
```

Comment:
Let's not do this. It hides the problem, not fixes it.

>>Panics are exceptional situations that can leave you app in a broken state. It may be better to let the app crash and restart it then to recover.